src/Workspaces/Core/Portable/CodeStyle/CodeStyleOption.cs

```diff
@@ -4,6 +4,8 @@
 
 #nullable disable
 
+#pragma warning disable RS0030 // Do not used banned APIs: CodeStyleOption<T>
```

Comment:
might be worthwhile having a comment explaining why it's ok to use here.  even if it's pedantic.  e.g. "this is the public API that we still ahve to support.  it will be removed/replaced yadda yadda".

Reply:
Will follow up.